hack/README.md

```diff
@@ -0,0 +1,58 @@
+docker/hack
+============================
+
+## About
+
+A collection of useful scripts that we use; we decided it deserves a great
+README! If there are any issues regarding the intention of a particular script
+(or even part of a certain script), please reach out to us. It may help us
+either refine our current scripts, or add on new ones that are appropriate for
+a given use case.
+
+**Note** None of these files are intended for manual execution. Please refer to
+https://docs.docker.com/opensource/project/test-and-docs/ which covers scripts
+in this folder running via `make`.
```

Comment:
if manual execution is not the intended purpose, then clarification below is not needed... and this should not be a 'note'.